rails/benchmarks/bm_scaffold_app.rb

```diff
@@ -0,0 +1,195 @@
+require 'bundler/setup'
+require 'rails'
+require 'action_controller/railtie'
+require 'active_record'
+require_relative 'support/benchmark_rails'
+
+class App
+  def self.request(method, path, body: {})
+    rack_input = StringIO.new(body.to_query)
+
+    env = {
+      "GATEWAY_INTERFACE" => "CGI/1.1",
+      "PATH_INFO" => path,
+      "QUERY_STRING" => "",
+      "REMOTE_ADDR" => "127.0.0.1",
+      "REMOTE_HOST" => "127.0.0.1",
+      "REQUEST_METHOD" => method.to_s.upcase,
+      "REQUEST_URI" => "http://localhost:3000#{path}",
+      "SCRIPT_NAME" => "",
+      "SERVER_NAME" => "localhost",
+      "SERVER_PORT" => "3000",
+      "SERVER_PROTOCOL" => "HTTP/1.1",
+      "SERVER_SOFTWARE" => "WEBrick/1.3.1 (Ruby/2.2.2/2014-05-08)",
+      "HTTP_HOST" => "localhost:3000",
+      "HTTP_ACCEPT" =>  "*/*",
+      "HTTP_USER_AGENT" => "HTTPie/0.8.0",
+      "rack.version" => [1, 2],
+      "rack.multithread" => false,
+      "rack.multiprocess" => false,
+      "rack.run_once" => false,
+      "rack.url_scheme" => "http",
+      "rack.errors" => StringIO.new,
+      "rack.input" => rack_input,
+      "CONTENT_LENGTH" => rack_input.length.to_s,
+      "CONTENT_TYPE" => 'application/x-www-form-urlencoded',
+      "HTTP_VERSION" => "HTTP/1.1",
+      "REQUEST_PATH" => path
+    }
+
+    _, _, body = ScaffoldAPP.call(env)
+    body.close if body.respond_to?(:close)
+  end
+end
+
+class ScaffoldAPP < Rails::Application
+  config.secret_token = "s"*30
+  config.secret_key_base = 'foo'
+
+  # Code is not reloaded between requests.
+  config.cache_classes = true
+
+  # Eager load code on boot. This eager loads most of Rails and
+  # your application in memory, allowing both threaded web servers
+  # and those relying on copy on write to perform better.
+  # Rake tasks automatically ignore this option for performance.
+  config.eager_load = true
+
+  # Full error reports are disabled and caching is turned on.
+  config.consider_all_requests_local       = false
+  config.action_controller.perform_caching = true
+
+  # Use the lowest log level to ensure availability of diagnostic information
+  # when problems arise.
+  config.log_level = :debug
+
+  # Send deprecation notices to registered listeners.
+  config.active_support.deprecation = :notify
+
+  # Use default logging formatter so that PID and timestamp are not suppressed.
+  config.log_formatter = ::Logger::Formatter.new
+
+  # otherwise deadlock occurs
+  config.middleware.delete "Rack::Lock"
```

Comment:
I disabled logs here with `NullLoger`, because speed of log writing may depend on SSD/HDD of the computer.
What do you think about disabling logs for benchmark?